{
    "type": "bundle",
    "id": "bundle--4adaa9ba-fac8-4cfa-9cc1-4cf86092b18c",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--50ac7bdc-1747-4042-bfa4-9926a834f938",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-04-11T00:23:07.238558Z",
            "modified": "2025-04-11T00:23:07.238558Z",
            "name": "CVE-2025-32699",
            "description": "Vulnerability in Wikimedia Foundation MediaWiki, Wikimedia Foundation Parsoid.This issue affects MediaWiki: before 1.39.12, 1.42.6, 1.43.1; Parsoid: before 0.16.5, 0.19.2, 0.20.2.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-32699"
                }
            ]
        }
    ]
}